catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/CsexraAttribute.java

```diff
@@ -396,15 +396,15 @@ private static Serializable getNiirsFunction(Tre tre) {
 
     Serializable value = TreUtility.getTreValue(tre, PREDICTED_NIIRS_SHORT_NAME);
 
-    if (value instanceof String && StringUtils.isNotEmpty((String) value)) {
+    if (value != null && StringUtils.isNotEmpty((String) value)) {
```

Comment:
❓ What does this change in this line do?

Reply:
`TreUtility.getTreValue` returns a string and the instance of check is redundant.